guides-for-using-cinchy/builder-guides/creating-tables/columns.md

```diff
@@ -194,47 +197,64 @@ Cinchy has several Date column type display format options available:
 ![](<../../../.gitbook/assets/image (81).png>)
 
 {% hint style="info" %}
-Please Note: the "Default Value" field isn't mandatory and should be left blank (best practice). However, if populated you won't be able to clear the default date value provided to a "blank" data (no date). You will only be able to overwrite it with another date value.
+The "Default Value" field isn't mandatory and should be left blank (best practice). However, if populated you won't be able to clear the default date value provided to a "blank" data (no date). You will only be able to overwrite it with another date value.
 {% endhint %}
 
-_These are equivalent to `DATE()` data type in SQL._
-
+{% hint style="info" %}
+These are equivalent to `DATE()` data type in SQL.
+{% endhint %}
 ### Yes/No
 
 You must select a default value of yes (1) or no (0) for yes/no fields.
 
-_These are equivalent to `bit`data type in SQL._
+{% hint style="info" %}
+These are equivalent to `bit`data type in SQL.
+{% endhint %}
+
+### Choice
 
-### Calculated
+You can create a choice column (single or multi-select) in Cinchy. In this scenario, you specify all your choices (1 per newline) in the table design. A user will only be able to select from the options provided.
+## Calculated Columns 
 
-A calculated column is evaluated using other fields on the record. It will also have a result type - which is the form in which the calculated results will be stored.
+A calculated column uses values from other fields in the record for its evaluation. These columns also have a specified result type, which dictates the format of the calculated output.
 
-For example, you can have a column `[Full Name]` that's `CONCAT([First Name], ' ', [Last Name])`_._
+**Example**:  
 
-_These are equivalent to computed columns in SQL._
+A `[Full Name]` column can be calculated as `CONCAT([First Name], ' ', [Last Name])`.
 
-#### Cached vs Live Calculated Columns
+{% hint style="info" %}
+These columns are similar to computed columns in SQL databases.
+{% endhint %}
+### Live vs Cached Calculated Columns
 
-When creating a calculated column, you will notice the option to have it cached or not, located under Advanced Settings. This was an option introduced in version 4.0 of the platform.
+#### Choose Your Calculation Type
 
-![](<../../../.gitbook/assets/image (304).png>)
+When creating a calculated column, you have two types to choose from: **cached** and **live**. This feature is accessible via the Advanced Settings and was a part of the 4.0 version update.
```

Comment:
Did we standardize on calling non-cached CCs "live"? 

We don't show that anywhere in the interface

Reply:
No idea. Who is the owner of the column types? Live calculated columns is also a mouthful.